generators/app/index.js

```diff
@@ -4,139 +4,177 @@
  * Drizzle Project Yeoman Generator
  *
  * @example yo drizzle
+ * @example yo drizzle --full
+ * @example yo drizzle --help
  */
 
 const chalk = require('chalk');
 const inquirer = require('inquirer');
 const yeoman = require('yeoman-generator');
-const yosay = require('yosay');
+const utils = require('./utils');
 
-const separator = label =>
-  new inquirer.Separator(`--- ${label} ---`);
+const INTRO = `
+${chalk.bold.underline('Welcome to your new Drizzle project.')}
+${chalk.dim('Beginning setup phase...')}
+`;
+
+// For use in a prompt
+const nodeVersions = [
+  '>=4.0.0',
+  '>=5.0.0'
+];
+
+// Command options (e.g. you drizzle --full)
+const standardOptions = new Map([
+  ['full', {
+    desc: 'Present the full list of prompts.',
+    type: Boolean,
+    default: false
+  }]
+]);
 
-const dependencyConflicts = new Map([
-  ['ramda', 'lodash'],
-  ['lodash', 'underscore'],
-  ['underscore', 'ramda']
+// Optional things to install
+const dependencies = new Map([
+  ['gsap', {type: 'js'}],
+  ['jquery', {type: 'js'}],
+  ['lodash', {type: 'js'}],
+  ['moment', {type: 'js'}],
+  ['ramda', {type: 'js'}],
+  ['postcss-easings', {type: 'postcss'}],
+  ['postcss-mixins', {type: 'postcss'}],
+  ['css-modularscale', {type: 'postcss'}]
 ]);
 
-const prompts = [
+// This is merged into props after the prompt phase.
+const computedProps = props => {
+  return {
+    repository: utils.toGitHubUrl(props.author, props.slug)
+  };
+};
+
```

Comment:
This could be re-written:

```
const computedProps = props =>
  ({ repository: utils.toGitHubUrl(props.author, props.slug) });
```

if you wanted (point being, wrapping object literal in parens will let you use the implicit return if you like).